predictions/multilabel_model/multilabel_pred_on_wavfile.py

```diff
@@ -0,0 +1,126 @@
+"""
+Testing a BR - Model
+"""
+import argparse
+import json
+import pickle
+import pandas as pd
+import numpy as np
+from tensorflow.compat.v1.keras import backend as K
+from predictions.binary_relevance_model import generate_before_predict_BR
+from predictions.multilabel_model import multilabel_pred
+
+def predict_on_embedding(embedding, config_datas):
+    '''
+    Predict on single embedding for audio clip
+    '''
+    prediction_probs = []
+    prediction_rounded = []
+
+
+    config_data = config_datas
+
+    # Clear keras session before all predictions with model
+    K.clear_session()
+
+    model = multilabel_pred.load_model(config_data["networkCfgJson"],
+                                                    config_data["train"]["outputWeightFile"])
+
+    ##############################################################################
+          # Predict on test data
+    ##############################################################################
+
+    if embedding.shape[0] < 10:
+        # Pad by repeating the last second in embedding till we get to 10 seconds
+        padding = np.ones((10-embedding.shape[0], embedding.shape[1]))*embedding[-1,:]
+        embedding = np.vstack([embedding, padding])
+
+    # If the clip is longer then 10 seconds, then predict on multiple 10-second
+    # windows within the clip. Take the max probability across all windows.
+    pred_prob = 0
+    for index in np.arange(10,embedding.shape[0]+1):
+        windowed_embedding = embedding[(index-10):index,:].reshape((1,10*128,1))
+        pred_prob = max(pred_prob, model.predict(windowed_embedding).ravel()[0])
+
+    # Clear keras session after all predictions with model
+    K.clear_session()
+
+    pred_round = np.round(pred_prob)
+    pred_prob = pred_prob * 100
+
+    prediction_probs.append(pred_prob)
+    prediction_rounded.append(pred_round)
```

Comment:
This block should give 7 outputs, instead it is giving one. The following suggestion fixes this.

```suggestion
    prediction_probs = np.zeros(len(config_data["labels"]))
    for index in np.arange(10,embedding.shape[0]+1):
        windowed_embedding = embedding[(index-10):index,:].reshape((1,10*128,1))
        window_pred_prob = model.predict(windowed_embedding)
        prediction_probs = np.array([prediction_probs, window_pred_prob.ravel()]).max(axis=0)

    # Clear keras session after all predictions with model
    K.clear_session()

    prediction_rounded = np.round(prediction_probs)
    prediction_probs = prediction_probs * 100
```